pkgs/applications/networking/instant-messengers/discord/default.nix

```diff
@@ -44,6 +52,11 @@ let
           "https://dl-canary.discordapp.net/apps/osx/${version}/DiscordCanary.dmg";
         sha256 = "0mqpk1szp46mih95x42ld32rrspc6jx1j7qdaxf01whzb3d4pi9l";
       };
+      development = fetchurl {
+        url =
+          "https://dl-development.discordapp.net/apps/osx/${version}/DiscordDevelopment.dmg";
+        sha256 = "sha256-F/OR0l1Rh/YQaR5V/kmSLNQPwvFeWj/NfgY8fUYg9fA=";
+      };
     };
     # Only PTB bundles a MachO Universal binary with ARM support.
     aarch64-darwin = { ptb = darwin-ptb; };
```

Comment:
Development supports aarch64-darwin, do you mind adding it? See the changes in #179379 for an example of how this should be done (since the DMG contains a Universal Binary).